app/models/state_changer/standard_mx_tube.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module StateChanger
+  # Handles the basic transitions of the {Tube::StandardMx} purpose tubes
+  # @note As of 201-10-01 only used for 'Standard MX' and 'Tag MX' tubes (Gatekeeper)
```

Comment:
what's '201-10-01' meant to be? :)

Reply:
Ah, I see that's not a new comment, it's copied

Reply:
I think it was supposed to be a date. I'll see if I can work it out.

Reply:
2019